articles/iot-edge/how-to-provision-devices-at-scale-linux-symmetric.md

```diff
@@ -132,7 +132,7 @@ Have the following information ready:
 
 1. Update the values of `id_scope`, `registration_id`, and `symmetric_key` with your DPS and device information.
 
-   The symmetric key parameter can accept a value of an inline key, a file URI, or a PKCS#11 URI. Uncomment just one symmetric key line, based on which format you're using.
+   The symmetric key parameter can accept a value of an inline key, a file URI, or a PKCS#11 URI. Uncomment just one symmetric key line, based on which format you're using. The inline key should be base64-encoded like the example, but if you're using a file URI the file should contain the raw bytes of the key.
```

Comment:
```suggestion
   The symmetric key parameter can accept a value of an inline key, a file URI, or a PKCS#11 URI. Uncomment just one symmetric key line, based on which format you're using. When using an inline key, use a base64-encoded key like the example. When using a file URI, your file should contain the raw bytes of the key.
```

Reply:
Thanks @PatAltimore! That is more clear. Professors Strunk and White ("Elements of Style") give you a thumbs-up too. 😁 By the way, is PKCS#11 also binary? I thought those are base64-encoded text files.